Controllers/schoolController.js

```diff
@@ -19,4 +19,39 @@ exports.createSchool = async (req,res) => {
             message:error.message
         })
     }
+}
+
+exports.getOneSchool = async (req,res)=>{
+    try {
+        const {id} = req.params
+        const school = await schoolModel.findById(id);
+
+        if (!school) {
+           return res.status(404).json({
+            message:"school does not exist"
+           }) 
+        }
+        res.status(200).json({
+            message:"successful",
+            data:school
+        })
+    } catch (error) {
+     res.status(500).json({
+        message:error.message
+     })   
+    }
+}
+
+exports.getAllSchool = async (req,res)=>{
+    try {
+        const school = await schoolModel.findById()
```

Comment:
your query is wrong here sir
```suggestion
        const school = await schoolModel.find()
```